chia/cmds/coin_funcs.py

```diff
@@ -176,7 +176,7 @@ async def async_combine(args: Dict[str, Any], wallet_client: WalletRpcClient, fi
 async def async_split(args: Dict[str, Any], wallet_client: WalletRpcClient, fingerprint: int) -> None:
```

Comment:
if `args` had been typed as `Dict[str, str]` it would have been easier to spot this bug, Are there examples of args that are not strings?

Reply:
yep, wallet id.

Reply:
> yep, wallet id.

only floats are strings, normal ints are ints.